content/en/methods/admin/domain_blocks.md

```diff
@@ -225,13 +228,35 @@ Authorized user is not allowed to perform this action, or invalid or missing Aut
 }
 ```
 
-##### 422: Unprocessable entity
+##### 422: Unprocessable entity - Missing Parameter
 
 The domain parameter was not provided
 
 ```json
 {
-	"error": "Validation failed: Domain can't be blank"
+  "error": "Validation failed: Domain can't be blank"
+}
+```
+
+##### 422: Unprocessable entity - Existing Domain Block
```

Comment:
Arguably this shouldn't be a 422 but instead a 409 conflict, I'm not sure how easy that would be to change though, since the 422 is existing behaviour.